templates/firefox-ios.xliff

```diff
@@ -3061,22 +3108,167 @@
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
     </header>
     <body>
+      <trans-unit id="TodayWidget.ClosePrivateTabsButton">
+        <source>Close Private Tabs</source>
+        <note>Close Private Tabs button label</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.ClosePrivateTabsLabelV2">
+        <source>Close
+Private
+Tabs</source>
+        <note>Close Private Tabs with new line on each word to fix any wrapping issues</note>
```

Comment:
Should we enforce wrapping here (and in other similar instances) to avoid any issues down the road?